### PR TITLE
Extend #7956 with one more test.

### PR DESCRIPTION
**Pull request by TheBlueMatt**

### CODE REVIEW

**Comment by laanwj** [comment]
Thanks!
(continues #7956)

File: src/test/dbwrapper_tests.cpp [hunk 1]
```diff
@@ -238,4 +238,90 @@ BOOST_AUTO_TEST_CASE(iterator_ordering)
...
+    }
+    StringContentsSerializer& operator+=(const StringContentsSerializer& s) { return *this += s.str; }
+
+    ADD_SERIALIZE_METHODS;
+
+    template <typename Stream, typename Operation>
+    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
+        if (ser_action.ForRead()) {
+            str.clear();
+            char c;
```
Comment (paveljanik) [inline src/test/dbwrapper_tests.cpp]: Please initialize this (as `clang` said 8):
```C++
char c = '\0';
```

**Comment by paveljanik** [comment]
Concept ACK

**Comment by sipa** [comment]
@TheBlueMatt Can you address https://github.com/bitcoin/bitcoin/pull/7992/commits/7be47549f6c46ffc92203b7e376ac536ca153e50#r61928804 ?

File: src/test/dbwrapper_tests.cpp (hunk 1 above)
Comment (TheBlueMatt) [inline src/test/dbwrapper_tests.cpp]: Fixed.

**Pull request merged**